### PR TITLE
add DamageType package

### PR DESCRIPTION
**Pull request by Cokemonkey11**
See https://github.com/peq/WurstScript/pull/471 for history and discussion

### CODE REVIEW

**Comment by Frotty** [comment]
see comment on old PR

**Comment by Cokemonkey11** [comment]
@Frotty there are 5 commits and my comment on old PR

https://github.com/peq/WurstScript/pull/471

File: wurst/systems/DamageType.wurst [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+    return false
+
+
+init
+    onEnter(() -> begin
+        GetTriggerUnit()..addAbility(          DAMAGE_TYPE_CHECK_ID)
+                        ..makeAbilityPermanent(DAMAGE_TYPE_CHECK_ID, true)
+
+        return false
```
Comment (Frotty) [inline wurst/systems/DamageType.wurst]: 💃

**Comment by Cokemonkey11** [comment]
@Frotty so gud

**Comment by Frotty** [comment]
Thanks 🍺

**Pull request merged**